wg-naming/README.md

```diff
@@ -0,0 +1,70 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+--->
+# Naming Working Group
+
+
+## Stakeholder SIGs
+* SIG Architecture
+* SIG Contributor Experience
+* SIG Docs
+
+
+
+## Organizers
+
+* Celeste Horgan (**[@celestehorgan](https://github.com/celestehorgan)**), CNCF
+* Stephen Augustus (**[@justaugustus](https://github.com/justaugustus)**), VMware
+* Zach Corleissen (**[@zacharysarah](https://github.com/zacharysarah)**), Linux Foundation
+
+## Contact
+- Slack: [#wg-naming](https://kubernetes.slack.com/messages/wg-naming)
+- [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-wg-naming)
+- [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/wg%2Fnaming)
+<!-- BEGIN CUSTOM CONTENT -->
+
+**The following section will be reworked and formalized as a charter once the
+Working Group has been approved by the Steering Committee.**
+
+### Goals
+
+- Evaluate language and naming choices within the Kubernetes organization, with
+  a specific initial focus of:
+  - Removing language which cannot be fully separated from racist, sexist or
+    homophobic connotations in cultural contexts outside of computer
+    programming
+  - Improving clarity of codebases and documentation by replacing idioms,
+    metaphors and slang specific to the English language
+- Create a list of terms to avoid and propose alternatives
+- Define a process by which any member of the Kubernetes organization can
+  propose a language recommendation, have them evaluated based on a rubric, and
```

Comment:
Since working groups disband, I think we'll also need to define long term ownership for the process (is it the sponsoring sigs?) and be sure to include the current leads of that group as approvers of the process.  Speaking of which, who will be the approvers of the process?

Reply:
> long term ownership

I think there will be some ongoing actions, hopefully very minimal, like keeping presubmit checks up-to-date. I don't think the exact structure needs to be figured out here, maybe just say that finding a home for ongoing tasks is an exit criteria for the WG.

Reply:
Long term ownership needs to reside with a long term group (SIG or committee of some kind). Agree with @lavalamp, doesn't need to be solved now, but should be called out as part of the exit criteria for the WG

Reply:
> > long term ownership
> 
> I think there will be some ongoing actions, hopefully very minimal, like keeping presubmit checks up-to-date. I don't think the exact structure needs to be figured out here, maybe just say that finding a home for ongoing tasks is an exit criteria for the WG.

I'm not looking to identify long term ownership in the charter, merely pointing out that the charter should acknowledge the need for long term ownership of the process and that the working group ought to present the proposed process to the long term owner for approval as opposed to self-approving.

Reply:
I think this is reasonable. I'll add a bullet point reading:

> "Define long-term ownership of this process"

Beyond that, I don't think it's worth diving into the weeds much more during the formation proposal. Sound good? :)